mono/metadata/filewatcher.c

```diff
@@ -43,11 +44,11 @@ ves_icall_System_IO_FSW_SupportsFSW (void)
 	return 1;
 }
 
-gboolean
+MonoBoolean
 ves_icall_System_IO_FAMW_InternalFAMNextEvent (gpointer conn,
-					       MonoString **filename,
-					       gint *code,
-					       gint *reqnum)
+											   MonoStringHandleOut filename,
```

Comment:
As I recall, but I can't find it now, this code is very suspicious, because the redeclaration of `FAMEvent` is not well motivated, and is incorrect. But oh well, separate matter.